7_react/react2/week1/vite-project/src/components/UserSearchBar.jsx

```diff
@@ -0,0 +1,38 @@
+import { useContext } from "react";
+import { userContext } from "./UserSection";
+
+function UserSearchBar () {
+  const { query, setQuery } = useContext(userContext);
+
+  return (
+    <div>
+      <h4>User Search</h4>
+
+      <div>
+        <form>
+          <label>
+            Search for user name:
+            <input
+              type="text"
+              placeholder="insert user"
+              style={{
```

Comment:
Try to avoid using inline styles when possible. Instead, consider moving the styles to a CSS file.

Reply:
Hey @aina21 
Yes, thank you.